### PR TITLE
[doc] :book: device placement configuration

### PR DESCRIPTION
**Pull request by YuanmingLeee**
1. Device placement documentation
2. Fix bug in device placement for Hysia Decoder

### CODE REVIEW

**Comment by YuanmingLeee** [comment]
@QimingAi Pls try test with docker to see if device placement can work under docker

**Review by HuaizhengZhang — APPROVED** [review]

**Comment by HuaizhengZhang** [comment]
LGTM

@QimingAi You may have a test.

File: README.md [hunk 1]
```diff
@@ -81,50 +81,34 @@ bash scripts/download-data.sh
...
-python start_model_servers.py
-
-# Run Django
-python manage.py runserver 0.0.0.0:8000
+Run with docker :whale:
+```shell script
+docker run --rm \
+  --gpus all -d -p 8000:8000 \
+  --mount source=server/config/device_placement.yml,target=/content/server/config/device_placement.yml \
+  hysia/hysia:v2o
```
Comment (QimingAi) [inline README.md:111]:
```
--mount type=bind,source=$PWD/server/config/device_placement.yml,target=/content/server/config/device_placement.yml \
```

Comment (QimingAi) [inline README.md:111]:
```
--mount type=bind,source=$PWD/server/config/device_placement.yml,target=/content/server/config/device_placement.yml
```

**Review by QimingAi — COMMENTED** [review]
Image in Docker Hub is updated

**Comment by QimingAi** [comment]
Shall we change the default choices in the V2O demo? It takes quite a long time to process a video if the user forget to change the models.

**Comment by HuaizhengZhang** [comment]
> Shall we change the default choices in the V2O demo? It takes quite a long time to process a video if the user forget to change the models.

Agree

**Pull request merged**